content/integrations/aws-secrets-manager.md

```diff
@@ -106,7 +106,7 @@ You need an [AWS access key and AWS secret key pair](https://docs.aws.amazon.com
  
 2. Generate Client Credentials
 
-   Create credentials for the client application to use to access the KES server. 
+   Create credentials the client application uses to access the KES server. 
```

Comment:
maybe
```suggestion
   Create credentials to allow the client application to access the KES server. 
```
but also this sentence feels a bit redundant? The heading just above does say "Generate Client Credentials"

Reply:
You're right, it's redundant. ✂️